src/tests/Loader/CollectibleAssemblies/ByRefLocals/Unloaded.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <OutputType>Library</OutputType>
-    <DisableProjectBuild>true</DisableProjectBuild>
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
```

Comment:
Library and RPI seem contradictory (though I think this is the case where Library is the default but one of our files overwrites it when RPI is set)

Reply:
I agree it's somewhat dirty but it was the easiest way to suppress the "duplicate project name" error - I can rename the two projects if you find that preferable but it will require changes to the test themselves as their source code hardcodes the assembly name. I'll be happy for any suggestions how to address this in a cleaner way.

Reply:
Oh.. can you confirm that this is what is happening - 

ByRefLocals and Statics both have (unrelated) libraries called Unloaded.  They work because they are marked RPI and get different directories and then msbuild copies their references into those directories.  (Otherwise, the merged test group would have two Unloaded libraries.)  However, the merged test group would still get two Unloaded libraries (the duplicates error), so this stops them from being included in that directory.  These are likely getting simple wrappers generated with zero actual tests in them (weird but harmless).

I'm definitely not sure about this, but perhaps they could be removed from the overall MergedWrapperProjectReference.  This may be a lurking problem if RPI is removed from the actual tests here (though maybe that's not possible due to the tests themselves) or if the output directory structure changes, so renaming one might be useful anyway.

Reply:
OK, thanks for your feedback, I agree with you that it's probably going to be cleanest to deduplicate the name of the Unloaded library, I'll do that as the next commit.

Reply:
Fixed in 14th commit.